src/google/protobuf/stubs/common.h

```diff
@@ -220,7 +225,13 @@ class FatalException : public std::exception {
 
 // This is at the end of the file instead of the beginning to work around a bug
 // in some versions of MSVC.
-using namespace std;  // Don't do this at home, kids.
+using std::istream;
```

Comment:
Add a TODO here? I think we want to get rid of these using statements in public header files as well.